src/transformers/tokenization_utils_base.py

```diff
@@ -1863,11 +1933,16 @@ def default_chat_template(self):
         """
         return (
             "{% for message in messages %}"
+            "{% if (message['role'] != 'assistant') %}"
             "{{'<|im_start|>' + message['role'] + '\n' + message['content'] + '<|im_end|>' + '\n'}}"
-            "{% endfor %}"
-            "{% if add_generation_prompt %}"
-            "{{ '<|im_start|>assistant\n' }}"
+            "{% elif (message['role'] == 'assistant')%}"
+            "{{'<|im_start|>' + message['role'] + '\n'}}"
+            "{% generation %}"
+            "{{message['content'] + '<|im_end|>'}}"
+            "{% endgeneration %}"
+            "{{'\n'}}"
```

Comment:
deleted. 
maybe i can put an example here `src/transformers/models/llama/tokenization_llama.py`? I thought it could be useful to have an example other than the doc

Reply:
Hmn, we're going to be removing all of the default chat templates, so I don't think that works! I think putting it in the docs is the only place that makes sense.